app/Http/Controllers/MessagesController.php

```diff
@@ -65,11 +65,11 @@ public function store(){
         return redirect('messages');
     }
 
-    public function show($id){
+    public function show($id) {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->